src/components/cart/CartModal.tsx

```
@@ -0,0 +1,74 @@
+        <Flex flexDirection="column">
+          <Layout>
+            <CartItemsContainer>
+              <Divider />
+              {cartItems.map((cartItem) => (
+                <>
+                  <ProductItemModalCard
+                    key={cartItem.id}
+                    product={cartItem.product}
+                  />
+                  <Divider />
+                </>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)